tests/js/spec/components/sidebar/sidebarDrodown/index.spec.jsx

```diff
@@ -1,21 +1,27 @@
-import {render} from 'sentry-test/reactTestingLibrary';
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
 
 import SidebarDropdown from 'sentry/components/sidebar/sidebarDropdown';
 import ConfigStore from 'sentry/stores/configStore';
 
 function renderDropdown(props) {
   const user = ConfigStore.get('user');
   const config = ConfigStore.get('config');
-  const organization = TestStubs.Organization();
+  const organization = TestStubs.Organization({role: 'member'});
+  const routerContext = TestStubs.routerContext([
+    {
+      organization,
+    },
+  ]);
   return render(
     <SidebarDropdown
       orientation="left"
       collapsed={false}
       user={user}
       config={config}
-      organization={organization}
```

Comment:
note this was a bug before

Reply:
I wonder why TypeScript was OK about it

Reply:
@Neo-Zhixing It's because it's a Javascript file (it's a test)

Reply:
ok lol i was blind